### PR TITLE
feat: update supported versions in security file

### PR DESCRIPTION
**Pull request by sdo-1A**
## Proposed change

Update supported Otter versions

### CODE REVIEW

**Review by matthieu-crouzet — APPROVED** [review]

**Review by cpaulve-1A — DISMISSED** [review]

File: SECURITY.md [hunk 1]
```diff
@@ -27,9 +27,9 @@ The following table provides the status of the Otter versions under support and
 
 | Otter version | Angular version | Support ends |
 |---------------|-----------------|--------------|
+| ^9.2.0        | ^16.0.0         | 2024-11-08   |
 | ^8.2.0        | ^15.0.0         | 2024-05-18   |
```
Comment (cpaulve-1A) [inline SECURITY.md:31]:
```suggestion
| ^8.3.0        | ^15.0.0         | 2024-05-18   |
```

**Review by mrednic-1A — DISMISSED** [review]

File: SECURITY.md (hunk 1 above)
Comment (matthieu-crouzet) [reply SECURITY.md:31]: ^8.x

**Review by cpaulve-1A — APPROVED** [review]

File: SECURITY.md (hunk 1 above)
Comment (sdo-1A) [reply SECURITY.md]: don't we only support the last minor of each major ?

**Review by matthieu-crouzet — DISMISSED** [review]

**Review by cpaulve-1A — APPROVED** [review]

**Review by fpaul-1A — APPROVED** [review]

File: SECURITY.md [hunk 2]
```diff
@@ -27,9 +27,9 @@ The following table provides the status of the Otter versions under support and
 
 | Otter version | Angular version | Support ends |
 |---------------|-----------------|--------------|
-| ^8.2.0        | ^15.0.0         | 2024-05-18   |
-| ^7.6.0        | ^14.0.0         | 2023-11-18   |
-| ^6.3.0        | ^13.0.0         | 2023-05-04   |
+| 9             | ^16.0.0         | 2024-11-08   |
```
Comment (kpanot) [inline SECURITY.md:30]: That's a bit weird to use 2 similar range notation for Otter versions and Angular version.
For me it should be `9 - 16` or `^9.0.0 - ^16.0.0`

**Review by matthieu-crouzet — APPROVED** [review]

**Review by fpaul-1A — APPROVED** [review]

**Pull request merged**